### PR TITLE
feat: dictionary support to core screens

### PR DESCRIPTION
**Pull request by tormodAase**
Supports english and norwegian

### CODE REVIEW

File: packages/core/src/components/screens/LoginScreen.tsx [hunk 1]
```diff
@@ -40,12 +42,13 @@ export const LoginScreen = () => {
                     onAuthenticationFailed={error =>
                         track(metricKeys.AUTHENTICATION, metricStatus.FAILED, undefined, { error })
                     }
+                    title={dictionary.login.logIn}
                     enableAutomaticAuthentication
                     scopes={authConfig.scopes || []}
                 />
                 {shouldDisplayDemoButton && (
                     <Button
-                        title="Demo"
```
Comment (HavardNot) [inline packages/core/src/components/screens/LoginScreen.tsx:48]: this is dedication

File: packages/core/src/components/screens/LoginScreen.tsx [hunk 2]
```diff
@@ -40,12 +42,13 @@ export const LoginScreen = () => {
...
+                        title={dictionary.login.demo}
```
Comment (HavardNot) [inline packages/core/src/components/screens/LoginScreen.tsx:51]: this is dedication

File: packages/core/src/language/nb.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        "locale": "Område"
+    },
+    "language": {
+        "language": "Språk",
+        "English": "Engelsk",
+        "Norwegian": "Norsk"
+    },
+    "settings": {
+        "title": "Innstillinger",
+        "aboutApp": "Om App'en",
```
Comment (HavardNot) [inline packages/core/src/language/nb.json:34]: I know this is probably just copied from mad-expo-core but this apostrophe really irks me for some reason :S

File: packages/core/src/language/useDictionary.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { useLanguage } from "../store/language"
+import * as enDictionary from "./en.json"
+import * as nbDictionary from "./nb.json"
+import { CoreDictionary } from "./types";
+
+export const useDictionary = (): CoreDictionary => {
+    const {language} = useLanguage();
+    switch (language.code) {
+        case "no":
+        case "nb":
```
Comment (HavardNot) [inline packages/core/src/language/useDictionary.ts:10]: Nynorsk confirmed TBD??? 🧨

**Review by HavardNot — DISMISSED** [review]
💥 💥 💥

File: packages/core/src/language/nb.json (hunk 3 above)
Comment (tormodAase) [reply packages/core/src/language/nb.json:34]: Yeah I'll remove it

File: packages/core/src/language/useDictionary.ts (hunk 4 above)
Comment (tormodAase) [reply packages/core/src/language/useDictionary.ts:10]: You never know 🇳🇴🇳🇴🇳🇴🇳🇴🇳🇴

**Review by HavardNot — APPROVED** [review]

**Pull request merged**